code/oled_block/oled.py

```diff
@@ -0,0 +1,53 @@
+import Adafruit_GPIO.SPI as SPI
```

Comment:
why not `from Adafruit_GPIO import SPI`?

Reply:
Now i  think that we don't need this library, we are not using spi to communicate with OLED